### ISSUE TITLE
 xy_loss的损失函数对应darknet 源码应该是　K.binary_crossentropy(true_xy - pred_xy)

### ISSUE THREAD

**Issue by luogantt** [zzh8829/yolov3-tf2#368]
xy_loss = obj_mask * box_loss_scale * \
            tf.reduce_sum(tf.square(true_xy - pred_xy), axis=-1)

**Comment by ZXTFINAL** [comment]
求导后的结果是一样的
